core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -131,7 +135,10 @@ private[storage] class BlockManagerDecommissioner(
                 // driver a no longer referenced RDD with shuffle files.
                 if (bm.migratableResolver.getMigrationBlocks(shuffleBlockInfo).size < blocks.size) {
                   logWarning(s"Skipping block $shuffleBlockInfo, block deleted.")
-                } else if (fallbackStorage.isDefined) {
+                } else if (fallbackStorage.isDefined
+                  // Confirm peer is not the fallback BM ID because then fallbackStorage would
+                  // already have been used in the try-block above and there's no point trying again
+                  && peer != FallbackStorage.FALLBACK_BLOCK_MANAGER_ID) {
```

Comment:
indentation?

Reply:
It looks correct to me. I also just tried running `./dev/scalafmt` and it used this indentation. Do you have another opinion on what it should be?

Reply:
Ya, I know that is a common mistake and we are mixing them frequently. @xkrogen .
We fallback to [Java convention](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html#248) if there is no rule in Scala style.

Your style falls into `//DON'T USE THIS INDENTATION` example exactly in Java convention. In short, Line 141 should not have the same indentation of line 140 for readability.

Reply:
Got it, thanks @dongjoon-hyun ! Fixed.